sdk/src/main/java/io/customer/sdk/CustomerIO.kt

```diff
@@ -538,6 +540,9 @@ class CustomerIO internal constructor(
             deviceRepository.addCustomDeviceAttributes(value)
         }
 
+    override val deviceToken: String?
```

Comment:
I thought we were going to use `currentlyRegisteredDeviceToken`? Or some other name that helps communicate that the value we return to you is one that you have registered with the SDK. `registeredDeviceToken` could be good, too.